Jenkinsfile

```diff
@@ -0,0 +1,6 @@
+#!groovy
+@Library('jenkins-jobs@kma-6-pipeline-deployments-for-scale-of-belief-lambda') _
```

Comment:
Is this library going to be shared across all the lambda deploys, or just scale-of-belief-lambda? Wondering if we should rename it.

Reply:
All the lambda deploys. However, I'll remove the `@kma-6-...` part of this line once https://github.com/CruGlobal/jenkins-jobs/pull/43 gets merged. It's a pointer to that branch of the jenkins-jobs repo, and isn't the name of this library.